packages/coordinator/src/components/Popup/Popup.tsx

```diff
@@ -0,0 +1,68 @@
+import { noop } from "lodash";
+import {
+  Dialog,
+  DialogTitle,
+  DialogActions,
+  DialogContent,
+  makeStyles,
+} from "@material-ui/core";
+import Button, { ButtonVariant } from "../Button";
+
+export type PopupProps = {
+  open: boolean;
+  title?: string | React.ReactNode;
+  content?: string | React.ReactNode;
+  onClose: (event: object, reason: string) => void;
```

Comment:
What is the `reason` used for?

Reply:
copied it from MUI `Dialog`'s signature, will remove